CHANGELOG.md

```diff
@@ -15,7 +15,10 @@
 - Added a [guide](docs/devctr-image.md) for updating the dev container image.
 - Added a new API call, `PUT /mmds/config`, for configuring the
   `MMDS` with a custom valid link-local IPv4 address.
-
+- Added JSON response format support for MMDS guest applications requests.
```

Comment:
We should specify that this is a feature we are currently experimenting with and is subject to change.

Reply:
I guess this would do it:
```suggestion
- Added experimental JSON response format support for MMDS guest applications requests.
```